### PR TITLE
Init

### PR DESCRIPTION
**Pull request by enxebre**
Provides first basic implementation

### CODE REVIEW

File: cloud/libvirt/actuators/machine/actuator.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+//// Copyright © 2018 The Kubernetes Authors.
```
Comment (frobware) [inline cloud/libvirt/actuators/machine/actuator.go]: The comment characters are doubled-up. s!////!//!g

File: cloud/libvirt/actuators/machine/actuator.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+// ActuatorParams holds parameter information for Actuator
+type ActuatorParams struct {
+	ClusterClient clusterclient.Interface
+}
+
+// NewActuator creates a new Actuator
+func NewActuator(params ActuatorParams) (*Actuator, error) {
+	return &Actuator{
+		clusterClient: params.ClusterClient,
+		counter:       0,
```
Comment (frobware) [inline cloud/libvirt/actuators/machine/actuator.go]: Default value is zero, so can drop this.

File: cloud/libvirt/actuators/machine/actuator.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	}, nil
+}
+
+// Create creates a machine and is invoked by the Machine Controller
+func (a *Actuator) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+	glog.Infof("Creating machine %v for cluster %v.", machine.Name, cluster.Name)
```
Comment (frobware) [inline cloud/libvirt/actuators/machine/actuator.go]: Nit: prefer %q for string values. Easier to select in the terminal.

File: cloud/libvirt/actuators/machine/actuator.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	a.counter++
+	if err := libvirtutils.CreateVolumeAndMachine(machine, a.counter); err != nil {
+		return fmt.Errorf("error creating machine %v", err)
+	}
+	return nil
+}
+
+// Delete deletes a machine and is invoked by the Machine Controller
+func (a *Actuator) Delete(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+	glog.Infof("Deleting machine %v for cluster %v.", machine.Name, cluster.Name)
```
Comment (frobware) [inline cloud/libvirt/actuators/machine/actuator.go]: Same as before; prefer %q for names/strings.

**Comment by enxebre** [comment]
thanks for the pointers @frobware I addressed your comments.
Let's get this in so we can a common place to collaborate and distribute tasks

**Pull request merged**